storage/aptosdb/src/lib.rs

```diff
@@ -1862,6 +1881,53 @@ impl DbReader for AptosDB {
         })
     }
 
+    // Returns latest `num_events` NewBlockEvents and their versions.
+    fn get_latest_block_events(&self, num_events: usize) -> Result<Vec<EventWithVersion>> {
+        gauged_api("get_latest_block_events", || {
+            if !self.skip_index_and_usage {
+                return self.get_events(
+                    &new_block_event_key(),
+                    u64::max_value(),
+                    Order::Descending,
+                    num_events as u64,
+                    self.get_latest_version().unwrap_or(0),
```

Comment:
again, I think we should take `ledger_version` as an input parameter

Reply:
I'd like block api talks block_height, and have the conversion between version <-> block to be a separate API. Added a TODO for adding block_height as parameter.